### PR TITLE
Update to Bootstrap 4.3 and fix related issues

### PR DESCRIPTION
**Pull request by jgolieb**
Update was required to get the spinner component which was introduced in 4.2 and we were on 4.1.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,22 +24,22 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bootstrap": "^4.1.3",
+    "bootstrap": "^4.3",
```
Comment (mihai-mihalache) [inline package.json:27]: can we please fix the bootstrap version? we don't want to have that upgraded without our knowledge.

Comment (mihai-mihalache) [reply package.json:27]: and I guess that goes the same for all our dependencies

**Review by mihai-mihalache — CHANGES_REQUESTED** [review]
We should also consider NOT using px() helper function all over the place. I think this should not be used where dimensions are supposed to be in pixels and not in rems (i.e. border width, etc.)

File: package.json (hunk 1 above)
Comment (jgolieb) [reply package.json:27]: Not sure what you mean. The purpose of this PR is to change the version to 4.3, so that's what was changed here.

**Comment by jgolieb** [comment]
I agree that the px() function should not be used every time we need to specify a length value. In many cases, we do not want the size of something to be relative to the font size, so we should use the px unit directly. I have actually changed from px() to just px in a number of places lately. If there is something specific in this PR that you think should be changed, please comment on it directly.

File: scss/components/_navbar.scss [hunk 2]
```diff
@@ -343,7 +343,7 @@
 
         .dropdown-toggle {
           position: absolute;
-          top: 0;
+          top: px(8);
```
Comment (mihai-mihalache) [inline scss/components/_navbar.scss:346]: maybe these could be used without px() helper function?

Comment (jgolieb) [reply scss/components/_navbar.scss:346]: OK, that probably makes sense. How about if I review the use of px() in _navbar.scss (which is where it it used most heavily) and submit any changes as a separate PR?

Comment (mihai-mihalache) [reply scss/components/_navbar.scss:346]: that would make most sense. as a matter of fact that's why I had that px() comment not tied to any one line of code

File: package.json (hunk 1 above)
Comment (jgolieb) [reply package.json:27]: Ah, I didn't get what you meant by "fix". I guess we should change to at least "~4.3", but I don't think using exactly "4.3" is necessary, do you?

**Review by mihai-mihalache — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (mihai-mihalache) [reply package.json:27]: I would do it just to be on the safe side. breaking changes happen all the time

Comment (mihai-mihalache) [reply package.json:27]: maybe we can address this too in another PR

**Pull request merged**